Kite SDK/PSPrintSDK/ViewController.m

```diff
@@ -35,6 +38,55 @@ -(void)viewDidAppear:(BOOL)animated{
 - (void)viewDidLoad {
     [super viewDidLoad];
     
+#ifndef DEBUG
+    self.navigationController.navigationBar.hidden = YES;
```

Comment:
I get why you're putting this in here but I wonder if it's the best idea. It kind of makes the sample app difficult to follow if you're coming to it as an integrating developer. Thoughts?